core/src/models/ContentGroup.ts

```diff
@@ -24,10 +25,11 @@ export interface IContentGroup<T extends ContentGroupType> {
      * - 'title' => 'inline' | 'pullout'
      * - 'orderedListItem' | 'unorderedListItem' => seperate index for each item (also for
      *      unordered items, otherwise items can't be distinguished)
+     * - 'line' => line number (needed to distinguish lines)
      *
      * - the requirement 'jesus words in red' can be achieved via quote='jesus'
      * - we have a seperate modifer for 'divineName' (in addition to 'person'), since person='god'
      *   would also match instances where divineName wouldn't
      */
-    modifier?: T extends 'title' ? 'inline' | 'pullout' : string;
+    modifier?: T extends 'title' ? 'inline' | 'pullout' : T extends 'line' ? number : string;
```

Comment:
This bit is a little confusing for me. 

Couldn't a content group have multiple modifiers? Why is it only possible for one modifier to be set?

Reply:
one group can only have one modifier. a phrase can have multiple group modifiers